--- postinst	2024-07-23 15:51:35.000000000 +0100
+++ postinst-openrc	2024-08-18 13:24:04.961013819 +0100
@@ -3,9 +3,9 @@
 
 chmod u+s "/usr/bin/mullvad-exclude"
 
-systemctl enable "/usr/lib/systemd/system/mullvad-daemon.service"
-systemctl start mullvad-daemon.service || echo "Failed to start mullvad-daemon.service"
-systemctl enable "/usr/lib/systemd/system/mullvad-early-boot-blocking.service"
+rc-update add mullvad-daemon default
+service mullvad-daemon start
+rc-update add mullvad-early-boot-blocking default
 
 # return 0 if version $1 is greater than or equal to $2
 function version_is_ge {
